README.md

```diff
@@ -1,42 +1,16 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
-
+# Rails API App
 
+>  The application is a an API that displays random greetings 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
+- Languages: Ruby
+- Framework: Ruby on Rails
+- Database: PostgreSQL
 
+## Link to Front-End App
+## [React App](https://github.com/NabilHY/hello-react-front-end)
 
```

Comment:
- [ ] Your front-end README file should provide a link to the Rails API (backend app). Kindly add this 👍🏻